ted_sws/notice_fetcher/adapters/ted_api.py

```diff
@@ -76,18 +79,19 @@ def get_by_query(self, query: dict) -> List[dict]:
         """
         query.update(DEFAULT_TED_API_QUERY)
 
-        response_body = api_request_with_query(api_url=self.ted_api_url, api_query=query)
+        response_body = self.request_api(api_url=self.ted_api_url, api_query=query)
 
         documents_number = response_body["total"]
         result_pages = 1 + int(documents_number) // 100
         documents_content = response_body["results"]
 
         for page_number in range(2, result_pages + 1):
             query["pageNum"] = page_number
-            response_body = api_request_with_query(api_url=self.ted_api_url, api_query=query)
+            response_body = self.request_api(api_url=self.ted_api_url, api_query=query)
             documents_content += response_body["results"]
         decoded_documents_content = []
         for document_content in documents_content:
+            print(f"Document content: {document_content['content']}")
```

Comment:
Let's start thinking of logging from the very start.

Reply:
Done